src/granite/querying.cr

```diff
@@ -28,13 +28,14 @@ module Granite::Querying
   # that you are using so you are not restricted or dummied down to support a
   # DSL.
   # Lazy load prevent running unnecessary queries from unused variables.
-  def all(clause = "", params = [] of Granite::Columns::Type)
+  def all(clause = "", params = [] of Granite::Columns::Type, use_primary_adapter = true)
```

Comment:
```suggestion
  def all(clause = "", params = [] of Granite::Columns::Type, has_performed_recent_write_action = true)
```

Reply:
This is addressed in the response to a similar change request.